src/participant_network.star

```diff
@@ -149,14 +149,32 @@ def launch_participant_network(
             network_params.deneb_fork_epoch,
             network_params.electra_fork_epoch,
         )
+    elif network_params.network in constants.PUBLIC_NETWORKS:
+        dummy = plan.run_sh(
+            run="mkdir /network-configs",
+            store=[StoreSpec(src="/network-configs/", name="el_cl_genesis_data")],
+        )
+        el_cl_data = el_cl_genesis_data.new_el_cl_genesis_data(
+            dummy.files_artifacts[0],
+            "0",
+        )
+
+        final_genesis_timestamp = 0
```

Comment:
do we not need these values for dora etc?

Reply:
Dora (and most other tools) have built in config for public testnets. 

I haven't bothered to do that yet, as didn't wanna get involved with gotemplating again.

Reply:
Cleaned up too.